zcash_client_backend/src/welding_rig.rs

```diff
@@ -35,29 +35,11 @@ fn scan_output<P: consensus::Parameters, K: ScanningKey>(
     params: &P,
     height: BlockHeight,
     index: usize,
-    output: CompactSaplingOutput,
+    output: CompactOutputDescription,
     vks: &[(&AccountId, &K)],
     spent_from_accounts: &HashSet<AccountId>,
     tree: &mut CommitmentTree<Node>,
-    existing_witnesses: &mut [&mut IncrementalWitness<Node>],
-    block_witnesses: &mut [&mut IncrementalWitness<Node>],
-    new_witnesses: &mut [&mut IncrementalWitness<Node>],
```

Comment:
I don't think it's worthwhile to document this, since the method is subsequently removed entirely; this was just split out as a separate commit to make the history easier to follow.